src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -273,6 +273,9 @@ export class SuggestController implements IEditorContribution {
 		// keep item in memory
 		this._memoryService.memorize(model, this.editor.getPosition(), item);
 
+		// keep line number for scrolling
+		const initialLineNumber = this.editor.getPosition().lineNumber;
```

Comment:
You should be using this `StableEditorScrollState`

Reply:
@jrieken 
Thanks for your review. I've update the PR.